startupscript/post-startup.sh

```diff
@@ -155,13 +199,13 @@ source "${SCRIPT_DIR}/bash-completion.sh"
 # git setup
 ###############
 if [[ "${LOG_IN}" == "true" ]]; then
-    source "${SCRIPT_DIR}/git-setup.sh"
+    retry 5 "${SCRIPT_DIR}/git-setup.sh"
 fi
 
 #############################
 # Mount buckets
 #############################
-source "${CLOUD_SCRIPT_DIR}/resource-mount.sh"
+retry 5 "${CLOUD_SCRIPT_DIR}/resource-mount.sh"
```

Comment:
Don't think we should retry resource-mount. Non-zero exit codes are expected, so we'd treat those as success.